.github/workflows/stale.yml

```
@@ -25,3 +25,4 @@ jobs:
         days-before-pr-stale: 45
         days-before-pr-close: -1
         operations-per-run: 1200
+        operations-per-run: 1300
```

actionlint (1.7.7)

key "operations-per-run" is duplicated in "with" section. previously defined at line:27,col:9. note that this key is case insensitive

(syntax-check)